### PR TITLE
Switch to uswds-jekyll

### LINKED ISSUE #128: switch to uswds-jekyll
See https://github.com/18F/content-guide/issues/234.

### PR DESCRIPTION
**Pull request by hoaiandang**
Fixes #128 
Navigation is currently not working

### CODE REVIEW

**Comment by hoaiandang** [comment]
Screen Shot 2020-08-02 at 9 05 34 PM

**Comment by afeld** [comment]
Seems to be there for me! (Maybe that local caching issue again?)

0 0 0 0_4000_future_problem_

@UchMok Thoughts about how to handle the long titles?

**Comment by hoaiandang** [comment]
There is a weird glitch that flashes when navigating between sites

**Comment by UchMok** [comment]
> Seems to be there for me! (Maybe that local caching issue again?)
> 
> 0 0 0 0_4000_future_problem_
> 
> @UchMok Thoughts about how to handle the long titles?

I've already changed title tense. So the titles should say:
- discover the current state
- define the future state
- deliver the right thing
- enable and empower partners

Looking like 27 max characters as of now, but I can shorten them if needed?

File: _config.yml [hunk 1]
```diff
@@ -23,107 +19,17 @@ incremental: true
...
-# Navigation
-# List links that should appear in the site sidebar here
-navigation:
-- text: Welcome
-  url: index.html
-  internal: true
-- text: Product Management at 18F
-  url: product-management-at-18f/
-  internal: true
-- text: Discover the current state
```
Comment (afeld) [inline _config.yml:44]: @hoaiandang Looks like this chunk of navigation is missing from the move. Mind re-copying?

**Review by afeld — APPROVED** [review]
Looks great! @UchMok We checked all the pages, but let us know if you see anything funky. Note the navigation configuration moved to `_data/navigation.yml`.

**Pull request merged**